### PR TITLE
rel note addition for node certificates

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to main and CP to  4.11 
- [Jira](https://issues.redhat.com/browse/OSDOCS-3516)
- Has Dev/QE ack.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Originated from #45643

File: windows_containers/windows-containers-release-notes-6-x.adoc [hunk 1]
```diff
@@ -21,22 +21,26 @@ ifndef::openshift-origin[]
...
 == Release notes for Red Hat Windows Machine Config Operator 6.0.0
 
-This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in 
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in
 
+=== New features and improvements
+[id="wmco-6.0.0-node-certificates"]
+==== Windows node certificates are updated
+
+With this release, the WMCO updates the Windows node certificates when the kubelet client certificate authority (CA) certificate is rotated.
```
Comment (bergerhoffer) [inline windows_containers/windows-containers-release-notes-6-x.adoc:40]: I think you need a blank line after this line. Some odd stuff with the leveloffset is showing in the build:

image

**Review by bergerhoffer — COMMENTED** [review]
One comment on the actual content

@Amrita42 It looks like this file is in main too. Can you please submit this update to main first, and then we'll CP to 4.11? You should be able to change the base of your existing local branch (and change it in the PR) by following the steps in the docs manual (see the section "What if I based my feature branch off of the wrong base branch?").

I'm out on PTO for a week after today, so please ping the peer review squad to follow up with merging after you've change it to go to main first. Thanks!

**Comment by Amrita42** [comment]
> @Amrita42: PR needs rebase.

Done a rebase follwing the steps in the manual

File: windows_containers/windows-containers-release-notes-6-x.adoc [hunk 2]
```diff
@@ -21,22 +21,27 @@ ifndef::openshift-origin[]
...
 
-If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support. 
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
 [id="wmco-6-0-0"]
 == Release notes for Red Hat Windows Machine Config Operator 6.0.0
 
-This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in 
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in
```
Comment (kalexand-rh) [inline windows_containers/windows-containers-release-notes-6-x.adoc:34]: Is there a missing line here?

Comment (Amrita42) [reply windows_containers/windows-containers-release-notes-6-x.adoc:34]: no actually the errata link will be added at a later stage when we have that available. thats the format we have for WMCO

**Pull request merged**